src/Console/ConfigurationResolver.php

```diff
@@ -150,10 +155,12 @@
     public function __construct(
         ConfigInterface $config,
         array $options,
-        $cwd
+        $cwd,
+        ToolInfo $toolInfo
```

Comment:
add missing `@param` for constructor

Reply:
This is conflicting with the config of the project. Shall I ajust `phpdoc_add_missing_param_annotation` to set `only_untyped` to `false`?

Reply:
is it conflicting? `$config` is in the docblock already

Reply:
You asked me to add a missing _typed_ `@param` while in our config we don't want to automatically add those.

Reply:
then I see conflict, as `@param $toolInfo` was not required, and `@param $config` is already there...